guestBook.txt

```diff
@@ -40,3 +40,5 @@ Tanju B - Hello! we finally making cuts down at the Maker Station makerspace in
 Maslow Bahrain (GeroBH) made it into the guestBook ;-) 21 Feb 2018
 
 Morriz was here - 9/5/2018
+
+esspe2 was here, working on linux,  hi from Belgium ! 28 Apr 2019
```

Comment:
It looks like the guest book change snuck in here too 😀

Reply:
Yeah, I have created the "correction" branch after the guest book change, so the modification was included, silly mistake ;-)

I'll see if I'm able to remove that file from this branch before the robot does the merge.
While I think that won't break anything.
And my other PR "Line added to the guestbook" should be merged before.
But some useful git practice btw.